storage/client_split_test.go

```diff
@@ -306,13 +306,14 @@ func TestStoreRangeSplitStats(t *testing.T) {
 	store, stopper := createTestStore(t)
 	defer stopper.Stop()
 
-	// Split the range at the first user key.
-	args := adminSplitArgs(proto.KeyMin, proto.Key("\x01"), 1, store.StoreID())
+	// Split the range after the last table data key.
+	keyPrefix := proto.Key("\xff\xfe")
```

Comment:
No, `TableDataPrefix.PrefixEnd()` is `\xff` and `Next()` on that key is `\xff\x00`.